core/src/test/java/org/pentaho/platform/util/StringUtilTest.java

```diff
@@ -75,14 +92,20 @@ public void testDoesPathContainParentPathSegment() {
     }
   }
 
-  public static void main( final String[] args ) {
-    StringUtilTest test = new StringUtilTest();
+  @Test
+  public void testMain() {
     try {
-      test.testTokenStringToArray();
```

Comment:
It seems that the testDoesPathContainParentPathSegment test was removed. Is this intentional?